core/src/main/java/com/sequenceiq/cloudbreak/EnvironmentVariableConfig.java

```diff
@@ -11,9 +11,9 @@
 
     public static final String CB_EVENTBUS_THREADPOOL_CORE_SIZE = "100";
 
-    public static final String CB_CERT_DIR = "/tmp/certs/";
-    public static final String CB_TLS_PRIVATE_KEY_FILE = "/tmp/certs/client-key.pem";
-    public static final String CB_TLS_CERT_FILE = "/tmp/certs/client.pem";
+    public static final String CB_CERT_DIR = "/certs/";
+    public static final String CB_TLS_PRIVATE_KEY_FILE = "/certs/client-key.pem";
```

Comment:
Could you do in that way that the below values do not expect a full path but they are composed from certDir + clientPrivateKey name and  certDir + clientCert
``` 
   @Value("${cb.tls.cert.file:" + CB_TLS_CERT_FILE + "}")
    private String clientCert;

    @Value("${cb.tls.private.key.file:" + CB_TLS_PRIVATE_KEY_FILE + "}")
    private String clientPrivateKey;
```